modules/core/src/main/java/org/apache/fluo/core/client/FluoAdminImpl.java

```diff
@@ -211,6 +212,10 @@ public void remove() {
         "The Zookeeper connection string (set by 'fluo.connection.zookeepers') "
             + " must have a chroot suffix.");
 
+    if (OracleServerUtils.oracleExists(getAppCurator())) {
```

Comment:
This is a very nice check.  Could drop `You` from error message `Must stop the oracle server to remove an application`